registration/auth_urls.py

```diff
@@ -31,9 +31,15 @@
 from django.contrib.auth import views as auth_views
 
 
+def login_user(request, template_name):
```

Comment:
I think it would be cleaner to move the `remember_me` functionality into a form, as django-allauth does. see https://github.com/pennersr/django-allauth/blob/cf45cc605a3d2b49efb1309960df7ee6059175cd/allauth/account/forms.py#L151